_data/docs.yml

```
@@ -46,6 +46,17 @@ apis:
       legacy: 1
       stable: 1
       nightly: 1
+  nx-cugraph:
+    name: nx-cugraph
+    path: cugraph # TODO: determine where to point nx-cugraph api docs
```

The docs are hosted as part of the cugraph docs, so this is correct.
```suggestion
    path: cugraph
```